Framework/Helpers/Waiters.cs

```diff
@@ -6,13 +6,12 @@ namespace Framework.Helpers
 {
     public class Waiters
     {
-
-        public static void WaitForPageToLoad(IWebDriver driver)
+        public static void WaitForPageToLoad(IWebDriver webDriver)
```

Comment:
Consider to rename this method to 'WaitForPageToLoadJs'

Reply:
Wow... Every method has JS interactions. Have you managed to make all of them work? If not, why did you leave them available for use?

Reply:
I do not use this helpers. Should I delete them&